papas/pfalgo/pfblocksplitter.py

```diff
@@ -29,8 +27,23 @@ def __init__(self,  block, unlink_edges, history_nodes = None):
         for edge in unlink_edges:
```

Comment:
I agree. Following the modified approach to blocks it needed some revision. I have simplified it and wonder whether it can now be ditched?